envoy/config/rbac/v2alpha/local_rbac.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.rbac.v2alpha;
+
+import "envoy/config/rbac/v2alpha/rbac.proto";
+
+// LocalRBAC specifies RBAC policies that apply to the local RBAC engine
+// that typically runs on a proxy. There can be multiple services running behind
+// a proxy.
+//
+// Here is an example of local RBAC policy, which contains policies for a single service.
+// {
+//   "products.default.svc.cluster.local": {
+//     policies: {
+//       "service-admin": {
+//         spec:
+//           rules:
+//           - services: [“*”]
+//             verbs: [“*”]
+//          bindings: {
+//            "bind-service-admin": {
+//              subjects:
+//              - user: “cluster.local/ns/default/sa/admin”
+//            },
+//          }
+//        },
+//        "product-viewer": {
+//          spec:
+//            rules:
+//            - services: [“products.default.svc.cluster.local”]
+//              paths: [“/products/*”, “*/reviews”]
+//              verbs: [“GET”, “HEAD”]
+//          bindings: {
+//            "bind-product-viewer": {
+//               subjects:
+//               - user: “alice@yahoo.com”
+//             },
+//           }
+//         },
+//       }
+//     },
+//  }
+message LocalRBAC {
```

Comment:
I still honestly don't really understand the concepts here. Can you define before the example:
- Fully qualified service name
- Service policy
- RoleInfo
- ServiceRole
- ServiceRoleBindings

Basically, here is my 99% use case: I want to add some RBAC policies on a route/vhost, or that match various things at the global request level. I don't really care about services, roles, etc. Can you clarify how I accomplish my very basic case and then build from there?

At the documentation level, I still am confused on how "role" is actually going to be used. How will the filter actually operate on this information?

Reply:
I agree with Matt. We seem to introduce too many concepts for basic things. For most people, they just need let x do y. It can be as simple as:
```
- paths: ["/products/*"]
  users: ["alice@yahoo.com"]
```